example/lib/download.dart

```diff
@@ -51,8 +51,8 @@ Future download2(Dio dio, String url, String savePath) async {
   }
 }
 
-void showDownloadProgress(received, total) {
+void showDownloadProgress(int received, int total) {
   if (total != -1) {
-    print((received / total * 100).toStringAsFixed(0) + '%');
+    print('${(received / total * 100).toStringAsFixed(0)}%');
   }
```

Comment:
```suggestion
   if(total < 1) return;
   final percentage = (received / total * 100).toStringAsFixed(0)
   print('$percentage%');
```

Reply:
- I replaced `1` with `0` because `0` is the actual boundary value.
- I merged the definition and the `print` statement together.

```diff
-  if (total < 1) return;
-  final percentage = (received / total * 100).toStringAsFixed(0)
-  print('$percentage%');
+  if (total <= 0) return;
+  print('percentage: ${(received / total * 100).toStringAsFixed(0)}%');
```

Reply:
> I replaced 1 with 0 because 0 is the actual boundary value.

That's fine for me.

> I merged the definition and the print statement together.

I believe that's harder to read, but since no one else complained, it seems like that's a me problem :D